xxhash.h

```diff
@@ -4002,8 +4004,8 @@ XXH3_len_129to240_64b(const xxh_u8* XXH_RESTRICT input, size_t len,
     #define XXH3_MIDSIZE_LASTOFFSET  17
 
     {   xxh_u64 acc = len * XXH_PRIME64_1;
-        int const nbRounds = (int)len / 16;
-        int i;
+        unsigned int const nbRounds = (unsigned int)len / 16;
```

Comment:
minor : any reason you prefer to employ `unsigned` instead of `int` for these variables ?

Reply:
Yeah, [unsigned seems to get better codegen](https://github.com/Cyan4973/xxHash/pull/754/commits/1456dc5cbf51ec810c44a7a17ed0faf8d17a3eaa) for GCC11/12.

Reply:
> Thanks @goldsteinn ! I can definitely confirm a notable bandwidth gains for the 17-128 variant of `xxh3` running on `x64` and compiled with `gcc`. The latency gain is much smaller, but it's also not a loss, so that's fine.
> 
> I would like to test on `arm64` | `m1` to confirm that the benefit is generic.

Great! Let me know if need to make it arch specific (although expect this helps any OOE machine).

Reply:
Yeah, likely the `/ 16` operation can only be safely translated into `>> 4` on `unsigned` types.